src/package_io/parse_input.star

```diff
@@ -219,6 +221,9 @@ def default_participant():
 def get_default_mev_params():
 	return {
 		"mev_relay_image": "flashbots/mev-boost-relay",
+		# TODO replace with actual when flashbots/builder is published
```

Comment:
can replace the TODO with "TODO replace with flashbots/builder when they publish an arm64 image" maybe tag the following issues

https://github.com/flashbots/builder/issues/107
https://github.com/flashbots/builder/issues/105

Reply:
We now have a multiarch builder going for flashbots builder image: https://github.com/ethpandaops/eth-client-docker-image-builder/pull/80